client.go

```diff
@@ -548,6 +548,82 @@ func (c *Client) SendInteractionResponse(ctx context.Context, interaction *Inter
 	return err
 }
 
+func (c *Client) CreateGlobalCommand(ctx context.Context, applicationID Snowflake, command *ApplicationCommand) error {
+	endpoint := fmt.Sprintf("/applications/%d/commands", applicationID)
+	req := &httd.Request{
+		Endpoint:    endpoint,
+		Method:      "POST",
+		Body:        command,
+		Ctx:         ctx,
+		ContentType: httd.ContentTypeJSON,
+	}
+	_, _, err := c.req.Do(ctx, req)
+	return err
+}
+
+func (c *Client) CreateGuildCommand(ctx context.Context, applicationID Snowflake, guildID Snowflake, command *ApplicationCommand) error {
+	endpoint := fmt.Sprintf("/applications/%d/guilds/%d/commands", applicationID, guildID)
+	req := &httd.Request{
+		Endpoint:    endpoint,
+		Method:      "POST",
+		Body:        command,
+		Ctx:         ctx,
+		ContentType: httd.ContentTypeJSON,
+	}
+	_, _, err := c.req.Do(ctx, req)
+	return err
+}
+
+func (c *Client) UpdateCommandGlobal(ctx context.Context, applicationID, commandID Snowflake, command *ApplicationCommand) error {
```

Comment:
The update struct might have to be adjusted to support default values: https://github.com/andersfylling/disgord/pull/420

Reply:
See how the fields are optional https://discord.com/developers/docs/interactions/application-commands#edit-global-application-command-json-params

Reply:
The default permission field will always return false if not set, and this can be unexpected behavior 
![image](https://user-images.githubusercontent.com/7851860/130367231-6df11984-75e4-4bb7-a5ad-1d72078c489e.png)

Reply:
The Application cmd struct also holds more fields than the Update endpoint allows / document.